aten/src/ATen/native/cuda/RangeFactories.cu

```diff
@@ -51,4 +65,34 @@ Tensor& linspace_cuda_out(Tensor& result, Scalar start, Scalar end, int64_t step
   return result;
 }
 
+Tensor& logspace_cuda_out(Tensor& result, Scalar start, Scalar end, int64_t steps) {
+  AT_CHECK(steps >= 0, "number of steps must be non-negative");
+
+  if (result.numel() != steps) {
+    result.resize_({steps});
```

Comment:
Huh, interesting that we're willing to write into any tensor that is correct numel. Well, I suppose it's handled correctly below.

Reply:
yes, it's strange but that was the existing behavior.